cvxpy/atoms/norm.py

```diff
@@ -69,6 +69,10 @@ def norm(x, p: Union[int, str] = 2, axis=None):
             return norm1(x, axis=axis)
         elif p in [np.inf, "inf", "Inf"]:
             return norm_inf(x, axis)
+        elif p == 'fro':
```

Comment:
```suggestion
        elif str(p).lower == "inf":
            return norm_inf(x, axis)
        elif str(p).lower == "fro":
```
`str(np.inf) == "inf"`